app/src/main/java/xyz/jekyllex/utils/Constants.kt

```diff
@@ -66,6 +66,13 @@ object Constants {
         "yaml" to "yml",
     )
 
+    val themeMap = mapOf(
+        1 to "one-light",
```

Comment:
Please change names to normal casing as per [original names](https://github.com/PrismJS/prism-themes/tree/master). `One Dark`, `Duotone Dark`, etc.